doc/release/0.4.0-notes.rst

```diff
@@ -47,6 +82,13 @@ Backwards incompatible changes
 arrays differing in size by one element may be produced, use the ``waverec``
 functions from the ``multilevel`` module instead.
 
+Bugs Fixed
+==========
+
+float32 inputs were not always respected. All transforms now return float32
+outputs when called using float32 inputs.
+
+incorrect detail coefficients were returned by downcoef when level > 1
```

Comment:
Maybe capitalize "incorrect" and use backticks around ``downcoef`` and ``level > 1``